content/rc/administration/setup/change-plan.md

```diff
@@ -7,22 +7,46 @@ categories: ["RC"]
 aliases: /rc/administration/setup-and-editing/changing-subscription-plan/
         /rc/administration/setup-and-editing/changing-subscription-plan/
 ---
-It is very easy to upgrade or downgrade your subscription at any time,
-all with no downtime.
+You can change your subscription plan at any time.
 
-To do this:
+To do so:
 
-1. Go to "Subscriptions" in the menu
-1. Click on the subscription name you want to edit
-1. Click on "Change Subscription" button at the bottom of the form
-1. Click on which plan you would like to upgrade or downgrade to
-1. Click the "Update" button
+1. From the Redis Cloud menu, choose **Subscriptions**.  
 
-Be aware that you cannot downgrade your plan if the databases in your
-subscription are larger in total size than the next lowest plan. For
-example, if you are currently on the 2.5GB plan and your databases total
-1.75GB, you would not be able to downgrade to the 1GB plan as your
-databases currently exceed the plan's limit. If you need to downgrade,
-you must remove data or databases to get below the threshold.
+    ![Select **Subscriptions** from the Redis Cloud menu](/images/rc/subscription-menu-select.png)
 
-{{< youtube uCcCnQ74C2U >}}
+1.  Select the subscription you want to change.
+
+    ![Select the target subscription from the list](/images/rc/subscription-list-select.png)
+
+1.  Select the **Change Description** button to list the available options.
+
+    ![The **Change subscription** button appears below the list of subscriptions.](/images/rc/subscription-change-button.png)
+
+
+1.  The first option you can change is type, which indicates whether the plan supports replication or data-persistence.  
+
+    Your current selection is highlighted.
+
+    ![The current plan type is selected.](/images/rc/subscription-type-cache.png)
+
+    To change this setting, select your preferred value.
+
+1.  Available memory sizes are displayed, along with prices.  
+
+    To change the memory size, choose your preferred value.
+
+    If you are upgrading a free plan, you need to select or add a payment method.
+
+1.  When finished, select the **Update** button.
+
+Before updating to a plan with a smaller memory size, verify that your data fits the new size.  Otherwise, the update process will fail and the plan will remain at the original size.
+
+
+Subscription updates happen in the background.  Changes may take several minutes to complete.  
+
+If the **Change Subscription** button is disabled, it usually means earlier updates are still deploying.
```

Comment:
This information should be located in step 3 where the user is selecting this button. That's where this error will occur.

Reply:
In my testing, the condition didn't occur in the simplest case (first use).  (It appeared during a second operation.)  Since it's not part of the "golden path" experience, I placed it at the end of the content-which is an appropriate place for irregular content.